homeworkweek3/src/components/laptops.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import Laptop from "./laptop";
+
+class Laptops extends Component {
+  constructor() {
```

Comment:
constructorul clasei tale, cat si apelul constructorului din clasa de baza (super()), trebuie sa contina ca parametru "props", altfel vei avea this.props undefined in constructor. Eu personal m-am obijnuit sa pun mereu acest parametru, pentru a nu "imi scapa vreodata" si a imi da batai de cap cu X bug-uri :D